ftw/simplelayout/handlers.py

```diff
@@ -70,3 +74,18 @@ def update_page_state_on_block_remove(block, event):
                         # Block has been removed
                         break
         config.store(page_state)
+
+
+def modify_parent_on_block_edit(block, event):
+    parent = aq_parent(block)
```

Comment:
should `aq_parent` always be used in combination with `aq_inner` :grey_question:
```python
parent = aq_parent(aq_inner(block))
```